### PR TITLE
the man with the 2 shotguns that blasted me to the end of the world with dualwielding. nerf

### PR DESCRIPTION
**Pull request by improvedname**
Anyways what this does.
This increases the shoot weight of improvised/bulldog shotguns so you cant fire them at the same time essiencetly making a instacrit pointblank or instacrit slugs

### CODE REVIEW

**Comment by WJohn** [comment]
What's the point of this when sawed offs exist??

**Comment by improvedname** [comment]
daulwielding shotguns with buckshots/rubbershots/shotgun slugs is not fun

**Comment by improvedname** [comment]
and are improvised/sawn off not subtypes from shotgun anyways

**Comment by WJohn** [comment]
Oh! Weapon weight! I misread that as w_class.

**Comment by improvedname** [comment]
sure do

**Comment by Kromgar** [comment]
Should have recoil if you dont have a hand open it blasts out of your hands and harms you

**Comment by WJohn** [comment]
thats a dumb snowflaky mechanic to implement just for shotguns.

**Comment by RandomMarine** [comment]
Literal i ded PR. No reasoning given on how this is supposed to improve the game.

**Comment by Supermichael777** [comment]
Wait no medium it you droped the gun because 2 stronk

File: code/modules/projectiles/guns/ballistic/revolver.dm [hunk 1]
```diff
@@ -263,6 +263,7 @@
 	icon_state = "dshotgun"
 	item_state = "shotgun"
 	w_class = WEIGHT_CLASS_BULKY
+	weapon_weight = WEAPON_MEDIUM
```
Comment (Supermichael777) [inline code/modules/projectiles/guns/ballistic/revolver.dm]: make it large and make sawnoff medium

**Comment by GeneralGoose** [comment]
i ded please nerf

**Comment by improvedname** [comment]
Ever tried avoiding 2 sprays buckshot pointblank

**Comment by improvedname** [comment]
And with dual wield is meant you can shoot them at the same time which even riot/compact/combat even cant do

**Comment by improvedname** [comment]
And BTW this isnt a i ded pr. I used this by myself accidently

File: code/modules/projectiles/guns/ballistic/revolver.dm (hunk 1 above)
Comment (improvedname) [inline code/modules/projectiles/guns/ballistic/revolver.dm]: This just make them unable to be used together

**Comment by PKPenguin321** [comment]
wait was this really merged

**Pull request merged**